libraries/CMSIS/prebuild_hook.py

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env python
```

Comment:
Can we specify in the project `rules.mk` the prebuild hooks we want to run? It might make it unnecessary for us to have as many empty hook files.